packages/client/src/sync/fetcher/reverseblockfetcher.ts

```diff
@@ -61,13 +64,11 @@ export class ReverseBlockFetcher extends BlockFetcher {
   }
 
   processStoreError(
-    error: Error,
+    _error: Error,
     _task: JobTask
   ): { destroyFetcher: boolean; banPeer: boolean; stepBack: bigint } {
     const stepBack = BIGINT_0
-    const destroyFetcher = !(error.message as string).includes(
-      `Blocks don't extend canonical subchain`
```

Comment:
@g11tech what is the reason to destroy the fetcher if there is another error? Not sure here :thinking:

Reply:
this is the only error that is expected if the peer didn't give you correct chain, for all other errors something wrong went in the fetcher and hence fetcher needs to be cleared out and a new fetcher will be restarted as a means to be robust against issues. why do you want to remove it?

Reply:
I should study the Fetcher stack a bit more before I can enlighten :) I read somewhere that if you destroy the fetcher it is a "critical" error, I would assume that then the fetcher is broken or something. Will study it some more and will get back to this later.

Reply:
yes, so fetcher will be/should be reinitiated (if our peer latest/best is correct or we handle it in better way)